### DISCUSSION TITLE
RDGeneral/export.h not found

### DISCUSSION THREAD

**Discussion by alphamol-basel** [rdkit/rdkit#7569] [Q&A] [answered]
Dear developers:

I am trying to use rdkit with C++ (GCC version 7.5.0 SUSE Linux).
The rdkit code was directly downloaded from https://github.com/rdkit/rdkit/tree/master/Code.
However, the file RDGeneral/export.h cannnot be found, although this file is cited by other files, e.g. GraphMol/GraphMol.h.

Thanks.

Describe the bug
RDGeneral/export.h file not found.

To Reproduce
rdkit.cpp:
#include <GraphMol/GraphMol.h>
#include

int main()
{
std::cout << "test" << std::endl;
return 0;
}

Compile command:
g++ -o test rdkit.cpp -I/home/shiyu/Documents/compression/rdkit/Code/

Output:
In file included from rdkit.cpp:1:0:
/home/shiyu/Documents/compression/rdkit/Code/GraphMol/GraphMol.h:15:10: fatal error: RDGeneral/export.h: No such file or directory
#include <RDGeneral/export.h>
^~~~~~~~~~~~~~~~~~~~
compilation terminated.

Expected behavior
To test whether i cite the rdkit code correctly.
Expected can print test string.

Configuration (please complete the following information):

RDKit version: 2024_03_3 Release.
OS: OpenSuse Linux 15.4
Are you using conda? None

**Comment by greglandrum** [comment]
Hi @alphamol-basel 
What commands did you use to build and install the RDKit?

**Reply by alphamol-basel** [reply, marked as answer]
Thanks for you reply!
Yesterday, I just download the Code folder in this Github repository instand of  installing the rdkit on my machine, and meet this problem.
Today, after you reminder, I install rdkit via anaconda and change the "include" path. 
The above problem has been solved.
However i still want to ask why the file RDGeneral/export.h cannnot be found in the code folder in this Github repository.
